3_javascript/javascript1/week4/script_homework_week4.js

```diff
@@ -0,0 +1,102 @@
+
+//Voice assistant - Create a function called getReply(command). The function should return a response that corresponds to the command!
+
+let getNameString = [];
+let getNameStringJustName = [];
+let toDoList = ['reading materials', 'doing exercises'];
+let getTimer = [];
+function getReply(Command) {
+
+    switch (Command) {
+        case 'Hello my name is Katayoun': //Command.include('Hello my name is') Katayoun': //should be changed to check for typing twice
```

Comment:
Nicely done, only thing is that the function works only with your name now :) it lacks elasticity, you can achieve it by using `.includes `method on the string, but you would need to replace the switch statements with` if else `